File: Programacion Multimedia/flutter/listacontactos/lib/Contacto.dart
// clase Contacto

class Contacto {
  String nombre = "";
  String apellidos = "";
  String edad = "";
  String telefono = "";
  String correo = "";
  String rutaImg = "";

  Contacto(String nombre, String apellidos, String edad, String telefono,
      String correo, String rutaImg) {
    this.nombre = nombre;
    this.apellidos = apellidos;
    this.edad = edad;
    this.telefono = telefono;
    this.correo = correo;
    this.rutaImg = rutaImg;
  }

  void set setNombre(String nombre) {
    this.nombre = nombre;
  }

  String get getNombre {
    return nombre;
  }
}

File: UI/NetBeansProjects/PracticaFinal/src/main/java/com/mycompany/practicafinal/Contacto.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.practicafinal;

/**
 *
 * @author FP
 */
public class Contacto {
    
    private String nombre;
     private String apellidos;
    private int edad ;
    private String url;

    public Contacto(String nombre, String apellidos, int edad) {
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.edad = edad;
    }

    public Contacto() {
    }

    public Contacto(String nombre, String apellidos, int edad, String url) {
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.edad = edad;
        this.url = url;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
    
    

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    @Override
    public String toString() {
        return "Contacto{" + "nombre=" + nombre + ", apellidos=" + apellidos + ", edad=" + edad + ", url=" + url + '}';
    }

    
    
    
    
   
    
}
